responses/es/00_class-introduction-issue.md

```diff
@@ -0,0 +1,31 @@
+## Paso 1: Activa GitHub Pages
+
+Bienvenida a GitHub Pages y Jekyll :tada:!
+
+Si es la primera vez que usas GitHub Pages o quieres aprender cómo crear y alojar un sitio web con [GitHub Pages](https://pages.github.com), estás en el lugar adecuado. Con GitHub Pages, puedes alojar contenidos como [documentación](https://flight-manual.atom.io/), [currículums](https://github.com/jglovier/resume-template) o cualquier otro contenido estático que quieras. 
```

Comment:
🍺 should we have `GitHub Pages` hyperlinked everywhere in the paragraph?

Reply:
I didn't look at that, just tried to translate text/links as they are. However I personally think it's enough to link the first instance.

Reply:
👍